starter-s3.yaml

```diff
@@ -169,69 +169,6 @@ data:
         containers:
         - image: alpine
           command: ["/bin/date"]
-    - cron: "17 * * * *"  # Every hour at 17 minutes past the hour
```

Comment:
where was the config come from? Did you see upstream one use similar setting so I can help check what's the problem?

Reply:
I actually don't suggest to use kubernetes cronjob because this will loose job tracking in Prow UI. that would be harder to diagnose problems. Everytime there's an issue, we need to login the cluster

Reply:
> where was the config come from? Did you see upstream one use similar setting so I can help check what's the problem?

The add file config is from [here](https://github.com/kubernetes/test-infra/blob/705997b53f349731aa03c355c50637af574a2917/label_sync/cluster/label_sync_cron_job.yaml)

> I actually don't suggest to use kubernetes cronjob because this will loose job tracking in Prow UI. that would be harder to diagnose problems. Everytime there's an issue, we need to login the cluster

Yes, I agree with you. I will look forward the original way to solve this problem.